libcontainer/standard_init_linux.go

```diff
@@ -186,6 +181,13 @@ func (l *linuxStandardInit) Init() error {
 			return newSystemErrorWithCause(err, "init seccomp")
 		}
 	}
+	if len(l.config.Args) == 0 {
```

Comment:
We want to validate this on creation not after we have done all this work.  Change the above code to if process == nil error out

Reply:
In other words, there is no reason to do 90% of container creation if we get to the end and we cannot run the container because process == nil

Reply:
> In other words, there is no reason to do 90% of container creation if we get to the end and we cannot run the container because process == nil

The point of this PR is allowing `create` for containers with no `process` (catching up with opencontainers/runtime-spec#701).  You can't `start` those containers, which is why we have this check here, but we certainly *don't* want to error those out during `create`.